### PR TITLE
Add monitor check hex edits for all IIDX LDJ games (tricoro+)

### PR DESCRIPTION
**Pull request by Poliwrath**
Run monitor check for 300 frames instead of 1200, and show 6 digits in monitor check instead of 4

### CODE REVIEW

**Comment by Poliwrath** [comment]
oki it's ready

File: copula.html [hunk 1]
```diff
@@ -10,83 +10,98 @@
...
-                          name : "Off",
-                          patch : [0x87, 0x3D, 0x11, 0x10],
-                         },
-                         {
-                          name : "Replace with song name",
-                          patch : [0x54, 0x1D, 0x42, 0x11],
-                         },
-                     ]
-                    },
+                new DllPatcherContainer([
```
Comment (mon) [inline copula.html]: This is still just one game and shouldn't be in a patch container

File: sinobuz.html [hunk 2]
```diff
@@ -101,83 +101,20 @@
...
-                         name : "Unlock All Songs",
-                         patches : [{offset : 0x5C125, off: [0x74, 0x15], on : [0x90, 0x90]}]
+                            //by nibs
+                            name: "Shorter monitor check",
+                            tooltip: "Runs for 300 frames (5 seconds) instead of 1200 (20 seconds), recommended only if you have a very stable framerate",
+                            patches: [{offset: 0x81A5A, off: [0xB0, 0x04], on: [0x2C, 0x01]}]
                         },
                         {
-                         name : "Cursor lock",
-                         tooltip : "After song finishes, song select remains on previous song",
```
Comment (mon) [inline sinobuz.html]: Can these really be merged?

Comment (Poliwrath) [reply sinobuz.html]: I don't know where these patches came from or what DLL it's for, but the SINOBUZ 7.1 omnimix DLL works fine with just the regular patches

File: copula.html (hunk 1 above)
Comment (Poliwrath) [reply copula.html]: Oh, I've been putting single games in patch containers in case a different version gets released

Comment (mon) [reply copula.html]: If you've been doing this and I haven't noticed, I shouldn't call it out here then.

**Comment by mon** [comment]
You're on the hook if that merge breaks someone's workflow <3

**Pull request merged**